lightly/utils/optim.py

```diff
@@ -76,8 +76,9 @@ def update_param_groups(
 
     # Update the optimizer's param_groups with the provided updates.
     for update in updates:
-        found_group = False
         name = update["name"]
+        found_group = False
+
```

Comment:
Please undo this change. It provides no value, but will mark the file and lines as last changed by this PR.